javascripts/components/chattyNav.js

```diff
@@ -1,6 +1,18 @@
-const users = ['Mohammad', 'Lance', 'Marco', 'Maggie'];
-      const select = document.getElementById('users');
+import { printToDom } from "../helpers/util.js";
 
-for (name in users){
-    select.add (new Option(users[name]));
-};
\ No newline at end of file
+const users = ["Mohammad", "Lance", "Marco", "Maggie"];
+
+const userPopulate = () => {
+  let domString = "";
+  domString += `<select class="selectpicker btn-success">`;
+  domString += `<option class="FirstItemHide">Select User</option>`;
+  for (let i = 0; i < users.length; i++) {
+    domString += `<option class="bg-light text-dark my-4" id="${
+      users[i]
+    }-user">${users[i]}</option>`;
```

Comment:
@m-khezri can you put all of this on one line? 
domString += `<option class="bg-light text-dark my-4" id="${users[i]}-user">${users[i]}</option>`;

Reply:
When I save the file VS-Code automatically arrange the code like this!! This is due to the extension Marco recommended to auto arrange codes after saving, Let me fix it...

Reply:
ha it's okay then. it will probably get reformatted again. i think it's fine!